backend/src/ledgers.ts

```diff
@@ -5,37 +5,102 @@ import { Response, NextFunction } from "express";
 
 import { redisClient, getOrThrow } from "./redisSetup";
 
-import { get30DayOldLedgerQuery, bqClient, BQHistoryLedger } from "./bigQuery";
+import { getBqQueryByDate, bqClient, BQHistoryLedger } from "./bigQuery";
 
-const LEDGER_DAY_COUNT = 30;
+export enum INTERVALS {
+  hour = "hour",
+  day = "day",
+  month = "month",
+}
+
+const LEDGER_ITEM_LIMIT = {
+  hour: 60,
+  day: 24,
+  month: 30,
+};
+
+const REDIS_LEDGER_KEYS = {
+  hour: "ledgers_hour",
+  day: "ledgers_day",
+  month: "ledgers",
```

Comment:
let's make this month to to be consistent
```suggestion
  month: "ledgers_month",
```